docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -91,15 +89,65 @@ for HTTP and WS.
 
 Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
 token if access is required after token expiration.     
+
+## JWT Public Key Authentication
+
+Enable authentication from the command line and supply the public key of the external JWT token. The JWT token must use the `RS256` algorithm.
```

Comment:
We should also mention that the login endpoint is disabled if using public key authentication

Reply:
Done

Reply:
Done.